mirrord/auth/src/credentials.rs

```diff
@@ -183,20 +176,58 @@ pub mod client {
     use super::*;
 
     impl Credentials {
-        /// Create `rfc2986::CertificationRequest` and send to operator to replace `Certificate`
-        /// inside of self.certificate making the Credentials ready
-        pub async fn get_client_certificate<R>(
-            &mut self,
+        /// Create a [`rfc2986::CertificationRequest`] and send it to the operator.
+        /// If the `key_pair` is not given, the request is signed with a randomly generated one.
+        pub async fn init<R>(
```

Comment:
Any special reason not to call this `new`?

Reply:
I guess not, I just left the old name :shrug: